### PR TITLE
Basic Tari Universe Provider

### PR DESCRIPTION
**Pull request by MCozhusheck**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "@tariproject/wallet_jrpc_client": "file:../tari-dan/clients/javascript/wallet_daemon_client",
+    "@tariproject/tarijs": "../tari.js",
```
Comment (kowalski) [inline package.json]: github link?

**Review by kowalski — APPROVED** [review]
How about we move this repo under `tari-project` organization?

File: src/App.tsx [hunk 2]
```diff
@@ -1,32 +1,24 @@
...
+  permissions,
+} from "@tariproject/tarijs";
 
-let __id = 0;
-
-type Request = {
-  id: number;
-  methodName: keyof TariProvider;
-  args: unknown[];
+const TUpermissions = new TariPermissions();
```
Comment (karczuRF) [inline src/App.tsx]:
```suggestion
const tuPermissions = new TariPermissions();
```

File: src/App.tsx [hunk 3]
```diff
@@ -1,32 +1,24 @@
...
+TUpermissions.addPermission(new permissions.TariPermissionKeyList());
```
Comment (karczuRF) [inline src/App.tsx]: Is there any reason to not add all permissions as array with single method?
```suggestion
tuPermissions.addPermissions([new permissions.TariPermissionKeyList() ...  ]); //... add here all needed
```

**Review by karczuRF — APPROVED** [review]

**Pull request merged**